src/methods/area.jl

```diff
@@ -71,23 +71,53 @@ computed slighly differently for different geometries:
 signed_area(geom) = signed_area(GI.trait(geom), geom)
 
 # Points
-area(::GI.PointTrait, point) = zero(typeof(GI.x(point)))
+area(::GI.PointTrait, point) = GI.isempty(point) ?
+    0 : zero(typeof(GI.x(point)))
 
 signed_area(trait::GI.PointTrait, point) = area(trait, point)
 
+# MultiPoints
+function area(::GI.MultiPointTrait, multipoint)
+    GI.isempty(multipoint) && return 0
+    np = GI.npoint(multipoint)
+    np == 0 && return 0
```

Comment:
Could we make area always return `Float64` for type stability?

Another option is to check the type once at the start, then push it through as the first argument to an `_area` method, and let users specify it too if they need to